portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +34,35 @@ function createCommentElement(userName, comment) {
   divElement.innerText = userName + ': ';
   divElement.appendChild(pElement);
   return divElement;
-}
\ No newline at end of file
+}
+
+function getLoggedInStatus() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    const loggedIn = json.loggedInStatus;
+
+    if (loggedIn){
+      const pElement = document.getElementById('display-user');
+      pElement.style.display = "block";
+      pElement.textContent = "Logged in as: " + json.userEmail;
+      document.getElementById('logout-link').href = json.logoutUrl;
+      document.getElementById('comment-form').style.display = "block";
+    }else
```

Comment:
nit: fix formatting - space after } and before {

Reply:
Done: dbe9632